src/markup/options.pug

```diff
@@ -7,6 +7,43 @@ html
 
     body
         main(id="search-tailor-options-panel")
+
+            -
```

Comment:
I'd love to be able to pull this data in directly from addonData.js, but haven't found a clean way to do so with the current build process.